hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/node/SCMNodeManager.java

```diff
@@ -1023,6 +1023,15 @@ public DatanodeUsageInfo getUsageInfo(DatanodeDetails dn) {
     try {
       usageInfo.setContainerCount(getContainerCount(dn));
       usageInfo.setPipelineCount(getPipeLineCount(dn));
+      
+      long reserved = 0L;
+      final DatanodeInfo di = nodeStateManager.getNode(dn);
+      for (StorageReportProto r : di.getStorageReports()) {
+        if (r.hasReserved()) {
+          reserved += r.getReserved();
+        } 
+      }
+      usageInfo.setReserved(reserved);
```

Comment:
Please create a helper function to return sum of reserved for the datanode, to make the abstraction level in this block more consistent.

```java
      usageInfo.setContainerCount(getContainerCount(dn));
      usageInfo.setPipelineCount(getPipeLineCount(dn));
      usageInfo.setReserved(getTotalReserved(dn));
```